scripts/version.js

```diff
@@ -0,0 +1,41 @@
+const args = require('args');
```

Comment:
Couldn't we just use `npm version` for this? e.g. `npm version patch|minor|major`